### ISSUE TITLE
VS Code Java Installer Error: spawn filepath containing non-ascii char

### ISSUE THREAD

**Issue by ewertonluna** [microsoft/vscode-java-pack#181]
env:
```
os: win32
arch: x64
JAVA_HOME: undefined
JDK_HOME: undefined
```
log:
```
[23:08:51] Download JDK
[23:11:00] Download JDK ... [Completed]
[23:11:00] Install JDK
[23:12:37] Install JDK ... [Completed]
[23:12:37] Install Java Extensions
[23:12:38] Error occurred
[23:12:38] Error: spawn C:\Users\Usu�rio\AppData\Local\Programs\Microsoft VS Code\bin\code.cmd ENOENT

```

**Comment by akaroml** [comment]
@eskibear please take a look.

**Comment by Eskibear** [comment]
We use node-winreg to search Windows Registry for VSCode installation details. Installation directory contains unicode chars, but was not correctly read by the lib. I believe that's the culprit. 

> The Microsoft Windows console isn't capable of handling UTF-8 encoded text unless you set it up properly. If you see weird question marks for certain characters, it's probhably a problem with the encoding.
By default the console is setup to use an encoding that suits the language of the Windows operating system installation. Windows uses codepages to specify encodings for the console. 

See: https://github.com/fresc81/node-winreg#processing-utf-8-data

**Comment by Eskibear** [comment]
@xuezha007 @ewertonluna @gitc-admin @JVDZMN 
I've released a new version v0.1.3 of the installer, please download it 👉[**HERE**](https://aka.ms/vscode-java-installer-win)👈 and give it a try, see if you can successfully go though the whole installation.
